vorestation.dme

```diff
@@ -1064,6 +1064,7 @@
 #include "code\game\objects\effects\temporary_visuals\projectiles\tracer.dm"
 #include "code\game\objects\items\antag_spawners.dm"
 #include "code\game\objects\items\apc_frame.dm"
+#include "code\game\objects\items\balls.dm"

```

Comment:
This should probably have _vr appended at the end just so we know what files are vorestation-specific and which are Polaris files. Icon files should get the same treatment.

Reply:
Done.